modules/eforms/pages/active-versions/index.adoc

```diff
@@ -1,27 +1,63 @@
+****
 
-Exceptionally, the active period has been extended for versions expiring in 2024 to allow for the effort of the first SDK upgrades for most eSenders. This means that notices can be submitted to TED until the end of activity but the SDK version will no longer be patched during the extended active period. 
+New versions of the SDK are released periodically, not only to accommodate regulatory changes, but also to correct and enrich the existing metadata. To maintain an overall consistency of the data collected through eForms notices, in terms of completeness and quality, it is in the general interest of the public procurement community to ensure that outdated versions of the eForms specification are withdrawn from active use for applications that create eForms notices. 
 
-The table below indicate the end of patches and of activity for SDK versions. The Preview environment has the same active range as the Production environment. 
+To encourage the timely adoption of the most recent version of the eForms specification, the Publications Office has established the following policy for the withdrawal of outdated versions of the SDK.
 
-NOTE: These tables are provided for information only, the information provided by CVS should always be considered the reference.
+The policy is based on the notion that every SDK version has a predefined lifespan which starts when the TED Central Validation Service (TEDCVS) starts accepting notices created using the new version (right after its release), and ends after a predefined period of activity has elapsed (at which point, TEDCVS will automatically stop accepting any further notices created using that version of the SDK). An SDK version is said to be "active" during the period that TEDCVS accepts notices based on it. 
 
-.Patched and Active versions of the SDK
+== Terminology
+
+SDK release date::
+  The date when a new version of the SDK is released.
+SDK expiry date::
+  The date beyond which an SDK version is no longer accepted by the TED Central Validation Service. 
+  This date is initially set for every SDK version to twelve (12) months after its release date.
+  The expiry date of an SDK version may be extended beyond the standard lifespan of twelve (12) months (see "SDK extended lifespan" below).
+Active SDK::
+  A version of the SDK is said to be "active" during the period that TEDCVS accepts notices based on it. This period is defined by the SDK release date and the SDK expiry date.
+SDK standard lifespan::
+  The standard lifespan of an SDK version is the first twelve (12) months after its release date.
+SDK extended lifespan::
+  To accommodate the needs of eSenders, the Publications Office may decide to extend the lifespan of an SDK version beyond its standard length. If the lifespan of an SDK version is extended beyond the standard lifespan of twelve (12) months, the version is said to be in its "extended lifespan". 
+Deprecated SDK::
```

Comment:
I don't think the term "deprecated" is useful. From the definition here, it's the same as "not active". And the term "deprecated" is not used anywhere else in this page.
In addition, "deprecated" usually means that the use is discouraged, but still allowed.

Reply:
I agree with Bertrand, deprecated means it exists, but is not recommended